pkg/apis/planetscale/v2/vitessshard_types.go

```diff
@@ -195,6 +195,11 @@ type VitessShardTabletPool struct {
 	// Typically, these are used to mount volumes defined through extraVolumes.
 	// These values are applied to both the vttablet and mysqld containers.
 	ExtraVolumeMounts []corev1.VolumeMount `json:"extraVolumeMounts,omitempty"`
+
+	// Annotations are key/value metadata pairs that can be attached to any Pod to select
+	// objects that satisfy certain conditions. This can be used for monitoring or access
+	// management tools that use tags for discovery.
+	Annotations map[string]string `json:"annotations,omitempty"`
```

Comment:
This new field isn't actually being used anywhere yet. We need to update `reconcile_tablets.go` to combine the annotation it injects there (`drain.SupportedAnnotation`) with the user-supplied annotations from this new field before sending them to `vttablet.Spec`. Make sure not to mutate the input map while doing that.